addons/binding/org.openhab.binding.autelis/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,568 @@
+			<channel id="temp-poolht" typeId="temp-poolht" />
+			<channel id="temp-spaht" typeId="temp-spaht" />
+			<channel id="temp-htstatus" typeId="temp-htstatus" />
+			<channel id="temp-poolsp" typeId="temp-poolsp" />
+			<channel id="temp-spasp" typeId="temp-spasp" />
+			<channel id="temp-pooltemp" typeId="temp-pooltemp" />
+			<channel id="temp-spatemp" typeId="temp-spatemp" />
+			<channel id="temp-airtemp" typeId="temp-airtemp" />
+			<channel id="temp-tempunits" typeId="temp-tempunits" />
+			<channel id="temp-htpump" typeId="temp-htpump" />
+			<channel id="pump-pump1" typeId="pump-pump1" />
+			<channel id="pump-pump2" typeId="pump-pump2" />
+			<channel id="pump-pump3" typeId="pump-pump3" />
+			<channel id="pump-pump4" typeId="pump-pump4" />
+			<channel id="pump-pump5" typeId="pump-pump5" />
+			<channel id="pump-pump6" typeId="pump-pump6" />
+			<channel id="pump-pump7" typeId="pump-pump7" />
+			<channel id="pump-pump8" typeId="pump-pump8" />
+			<channel id="chlor-chloren" typeId="chlor-chloren" />
+			<channel id="chlor-poolsp" typeId="chlor-poolsp" />
+			<channel id="chlor-spasp" typeId="chlor-spasp" />
+			<channel id="chlor-salt" typeId="chlor-salt" />
+			<channel id="chlor-super" typeId="chlor-super" />
+			<channel id="chlor-chlorerr" typeId="chlor-chlorerr" />
+			<channel id="chlor-chlorname" typeId="chlor-chlorname" />
+			<channel id="lightscmd" typeId="lightscmd" />
+		</channels>
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<context>network-address</context>
+				<label>Host or IP</label>
+				<description>The host name or IP address of the Autelis Controller.</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="port" type="integer" min="1" max="65335"  required="true">
+				<context>network-address</context>
+				<label>Port</label>
+				<description>The port the Autelis Controller is listening on.</description>
+				<default>80</default>
+				<required>true</required>
+			</parameter>
+			<parameter name="user" type="text" required="true">
+				<label>User Name</label>
+				<description>The user name to use when connecting to a Autelis Controller.</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="pass" type="text" required="true">
+				<label>Password</label>
+				<description>The password to use when connecting to a Autelis Controller.</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="refresh" type="integer" required="true">
+				<label>Refresh interval</label>
+				<description>Specifies the refresh interval in seconds</description>
+				<default>5</default>
+				<required>true</required>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<channel-type id="system-runstate">
+		<item-type>Number</item-type>
+		<label>Runstate</label>
+		<description>Gets the controller's runstate. 1 = Starting Up, 2-49 = Getting Data
```

Comment:
actually, what if I mark it as an advanced feature, it might be useful to see where the controller is in its communication process or set a rule based on its value, but most people won't want to see it in the UI.